modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2013,6 +2050,39 @@ describe('V2 Wallet:', function () {
         args[1]!.should.equal('full');
       });
 
+      it('should call prebuildTxWithIntent with the correct params for eth fillNonce for receive address nonce filling tx', async function () {
+        const feeOptions = {
```

Comment:
Just checking whether this will work with legacy fee options also for BSC transfers?

Reply:
Based on this [method](https://github.com/BitGo/bitgo-microservices/blob/b0b1e2d4fdd8937674ef50d33cd594600dcb432f/packages/wallet-platform/app/controllers/api/v2/coins/abstractEthLikeCommon.ts#L2053) in platform, it seems like it should work for legacy fee options as well for BSC.